pkgs/by-name/ri/rimgo/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  fetchFromGitea,
+  buildGoModule,
+  tailwindcss,
+}:
+buildGoModule rec {
+  pname = "rimgo";
+  version = "1.2.0";
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "rimgo";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-C878ABs978viVtIuv3fPn2F2anOg2GB/+f5jaCO13tc=";
+  };
+  preBuild = ''
+    ${tailwindcss}/bin/tailwindcss -i static/tailwind.css -o static/app.css -m
+  '';
+  vendorHash = "sha256-u5N7aI9RIQ3EmiyHv0qhMcKkvmpp+5G7xbzdQcbhybs=";
+  meta = with lib; {
+    description = "An alternative frontend for Imgur";
```

Comment:
- put this in alphabetical ordering
- set `mainProgram` accordingly

Reply:
I thought `mainProgram` is only important if the main executable is different from the attribute name.
Is there any other use for `mainProgram` other than `nix run`, that can't default to the attribute name?

Reply:
good point with the alphabetical ordering tho

Reply:
Recently we discovered that this "default" never existed to begin with.

https://github.com/NixOS/nixpkgs/pull/246386

Since then, we are suggesting to set `meta.mainProgram`.